src/components/store/AddCartButton/style.module.scss

```
@@ -44,8 +49,8 @@
     border-radius: 10px;
     font-size: vars.$input-text-medium;
     font-weight: bold;
-    height: 40px;
-    width: 200px;
+    padding: 0.3rem 1rem;
+    word-break: break-all;
```

it's better to use

```suggestion
    word-break: break-word;
```

`break-all` ignores words and breaks wherever it wants, even when it's not necessary to break a word (fine for Chinese, not great for English), while `break-word` only breaks words if it's absolutely necessary

[here's a demo/documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/word-break)